presentation/components/src/presentation-components/instance-filter-builder/Utils.ts

```diff
@@ -144,6 +142,39 @@ function getCategorizedFieldName(fieldName: string, categoryName?: string) {
   return `${categoryName ?? ""}${INSTANCE_FILTER_FIELD_SEPARATOR}${fieldName}`;
 }
 
-function getPrefixedFieldName(str: string, prefix?: string) {
-  return prefix !== undefined ? `${prefix}${FIELD_NAMES_SEPARATOR}${str}` : str;
+function getPrefixedFieldName(str: string, prefix: string) {
+  return `${prefix}${FIELD_NAMES_SEPARATOR}${str}`;
+}
+
+/** @alpha */
+export function convertPresentationFilterToPropertyFilter(descriptor: Descriptor, filter?: PresentationInstanceFilter): PropertyFilter | undefined {
+  if (!filter)
+    return undefined;
+  return PresentationFilterToPropertyFilter(filter, descriptor);
+}
+
+/** @alpha */
+function PresentationFilterToPropertyFilter(filter: PresentationInstanceFilter, descriptor: Descriptor): PropertyFilter | undefined {
+  if (isPresentationInstanceFilterConditionGroup(filter)) {
+    const rules: PropertyFilter[] = [];
```

Comment:
Maybe it would be easier to read this code if conversion would be extracted to separate functions: `convertPresentationInstanceFilterConditionGroup` and `convertPresentationInstanceFilterCondition`